<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Statis</class>
 <widget class="QDialog" name="Statis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1001</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:#96969696;

</string>
  </property>
  <widget class="QLabel" name="stat1_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white
</string>
   </property>
   <property name="text">
    <string>Statistics:</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_5" native="true">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>180</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);
background-image: url(:/img/ressources/rouge.jpg);</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_7" native="true">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>510</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);
background-image: url(:/img/ressources/rouge.jpg);</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_6" native="true">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>150</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 0);
background-image: url(:/img/ressources/vert.jpg);</string>
   </property>
  </widget>
  <widget class="QLabel" name="stat3">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>450</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>450</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/img/ressources/blue.jpg);
background-color: rgb(0, 0, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>180</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>120</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/img/ressources/blue.jpg);
background-color: rgb(0, 0, 255);</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_8" native="true">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>480</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 0);
background-image: url(:/img/ressources/vert.jpg);</string>
   </property>
  </widget>
  <widget class="QLabel" name="stat1">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>510</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>150</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="stat2">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>480</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>TextLabel </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>120</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">text:white</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
